.github/workflows/build.yaml

```diff
@@ -15,41 +15,66 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
-      - name: Set up Python
-        uses: actions/setup-python@v2
+      - uses: conda-incubator/setup-miniconda@v2
         with:
           python-version: 3.8
-
-      - name: Install sqlite
-        run: sudo apt-get install sqlite libyaml-dev
+          channels: conda-forge,defaults
+          channel-priority: strict
+          show-channel-urls: true
 
       - name: Update pip/wheel infrastructure
         run: |
-          python -m pip install --upgrade pip
-          pip install wheel
+          conda install -y -q pip wheel
+
+      - name: Install sqlite
+        shell: bash -l {0}
```

Comment:
`bash -l` is needed to activate conda environment (via `.profile`)

Reply:
Because I missed it? Thanks for noticing, I'll fix it. And I'm not quite sure where does it install if you don't activate it, maybe in `lib/python` instead of separate environment. Would be much easier if I could run an interactive shell to look at things.